packages/nextjs/pages/unlock/partner/landing.tsx

```diff
@@ -0,0 +1,160 @@
+        </div>
+
+        {/* PRICE */}
+        <input
+          type="number"
+          min="1"
+          placeholder={PRICE_PLACEHOLDER}
+          className="input font-bai-jamjuree w-full px-5 bg-[url('/assets/gradient-bg.png')] bg-[length:100%_100%] border border-primary text-lg sm:text-2xl placeholder-white uppercase"
+          onChange={e => setPrice(Number(e.target.value))}
+          onKeyUp={e => e.key === "Enter" && handleOnSubmit()}
+          value={price}
+        />
+        {/* DURATION */}
+        <div className="grid grid-cols-3 gap-4">
+          <div className="col-span-1 flex items-center">
+            <label htmlFor="select" className="text-lg font-medium text-gray-600">
+              {SUBSCRIPTION_DURATION_LABEL}
+            </label>
+          </div>
+
+          <div className="col-span-2">
+            <div className="flex">
+              <select
+                id="select"
+                className="block w-full px-4 py-2 leading-5 text-gray-700 border rounded-md focus:outline-none focus:border-blue-500 focus:ring focus:ring-blue-200"
+                onChange={e => setDuration(e.target.value)}
+                value={duration}
+              >
+                <option value="1Month">1 Month</option>
+                <option value="3Months">3 Months</option>
+                <option value="9Months">9 Months</option>
+                <option value="12Months">12 Months</option>
+              </select>
+            </div>
+          </div>
+        </div>
+
+        {/* TOKEN */}
+        <div className="grid grid-cols-3 gap-4">
+          <div className="col-span-1 flex items-center">
+            <label htmlFor="select" className="text-lg font-medium text-gray-600">
+              {TOKEN_SELECTION_LABEL}
+            </label>
+          </div>
+
+          <div className="col-span-2">
+            <div className="flex">
+              <select
+                id="select"
+                className="block w-full px-4 py-2 leading-5 text-gray-700 border rounded-md focus:outline-none focus:border-blue-500 focus:ring focus:ring-blue-200"
+                onChange={e => setToken(e.target.value)}
+                value={token}
+              >
+                {/* APECoin is giving out prizes */}
+                <option value="USDC">USDC</option>
+                <option value="DAI">DAI</option>
+                <option value="USDT">USDT</option>
+                <option value="APE">APECoin</option>
+              </select>
+            </div>
+          </div>
+        </div>
+      </div>
+      {!isConnected && (
+        <div className="mt-8 flex flex-col sm:flex-row items-start sm:items-center gap-2 sm:gap-5">
+          <div className="flex rounded-full border border-primary p-1 flex-shrink-0">
+            <div className="flex rounded-full border-2 border-primary p-1">
+              <Web3AuthConnectButton></Web3AuthConnectButton>
```

Comment:
nitpick: improvement opportunity

```suggestion
              <Web3AuthConnectButton/>
```